src/main/java/ca/corefacility/bioinformatics/irida/ria/web/oauth/RemoteAPIController.java

```diff
@@ -302,7 +222,17 @@ public String checkApiStatus(@PathVariable Long apiId) {
 	 */
 	@RequestMapping("/status/web/{apiId}")
 	public String checkWebApiStatus(@PathVariable Long apiId) {
-		String status = checkApiStatus(apiId);
+		// TODO: This is to be removed when remote api details page is refactored
+		//       Use similar functionality in RemoateAPIAjaxController.
+		RemoteAPI api = remoteAPIService.read(apiId);
```

Comment:
Typo in `RemoateAPIAjaxController`

Reply:
Typo fixed in d5fc6cea95acf73168e620cfbc21ef9e47cdd04a